src/test/java/lesson13BubbleSort/BubbleSorterTest.java

```diff
@@ -0,0 +1,34 @@
+package lesson13BubbleSort;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.ListIterator;
+
+class BubbleSorterTest {
+
+    @Test
+    void shouldSort() {
+        //given
+        List<Integer> list = Arrays.asList(7,1,0,13,122,15);
+        //when
+        BubbleSorter.sort(list);
+        ListIterator<Integer> iterator = list.listIterator();
+        boolean result = true;
+        while(true){
```

Comment:
Po co te wszystkie obliczenia?

Wystarczy:  Assertions.assertEquals(list, List.of(0,1,7,15,122));